packages/relay/src/routes/user.ts

```diff
@@ -0,0 +1,53 @@
+import { Express } from 'express'
+import { DB } from 'anondb/node'
+import { Synchronizer } from '@unirep/core'
+import TwitterClient from '../singletons/TwitterClient'
+import crypto from 'crypto';
+
+const STATE = crypto.randomUUID()
+
+export default (app: Express, db: DB, synchronizer: Synchronizer) => {
+    app.get('/api/login', async (req, res) => {
+        res.redirect(TwitterClient.authClient.generateAuthURL({
+            state: STATE,
+            code_challenge: crypto.randomUUID()
+        }))
+    })
+
+    app.get('/api/callback', async (req, res) => {
```

Comment:
Should this name as callback or user as the draft api flow suggest?
![image](https://github.com/social-tw/social-tw-website/assets/35304302/628d9ea9-89f6-400c-9dba-f2b210a09715)